cli/tests/unit/console_test.ts

```diff
@@ -722,7 +722,6 @@ unitTest(function consoleTestWithCustomInspectorError(): void {
   class A {
     [customInspect](): string {
```

Comment:
The return type should be updated to `never` but should still pass type checking.

Reply:
Absolutely right! I'll fix it.